WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -395,11 +455,23 @@ private extension AppSettingsViewController {
     func mediaTableSection() -> ImmuTableSection {
         let mediaHeader = NSLocalizedString("Media", comment: "Title label for the media settings section in the app settings")
 
+        let imageOptimizationValue = Bool(MediaSettings().imageOptimizationSetting)
```

Comment:
Please, use a multiplication sign (`×`) to display dimensions.

Reply:
I opened a PR https://github.com/wordpress-mobile/WordPress-iOS/pull/22019. I also added thin spaces.